source/scss/variables/_custom-variables.scss

```diff
@@ -37,15 +37,27 @@
     Logo
 */
 
-
-
-// Z-INDEXES
-$backdrop-z-index: 1;
-$header-logo-z-index: 2;
-$cookie-alert-z-index: 100;
-$header-z-index: 10;
-$header-sub-menu-desktop-z-index: 1;
-$select-z-index: -1;
+// Z-INDEX MAP
+// @description: https://confluence.fti-group.com/display/PORTALS/z-index+strategy
+$z-layers: (
+  "bottomless-pit": (
+    "select-tab": -1
+  ),
+  "component": (
+    "base": 0,
+    "backdrop": 1,
+    "header-logo": 1,
+    "header": 10,
+  ),
+  "navigation": (
+    "base": 2000,
```

Comment:
Correct me, if I'm wrong, but wouldn't the navigation be over the backdrop? And also the notifications... Is this really wanted?

Reply:
+1

Reply:
Yes. The navigation was over the backdrop before as well. So it is correct, that it goes over the backdrop.
The backdrop is the element, that makes the background go grey. You can click there to close the navigation. It has a fixed position, with left and top 0, so really sticking onto the window and does not follow its DOM flow. But it is placed under the navigation. Let's call this part sub-navigation, because this is what it is supporting.

Since we have only a tiny bit of playground, most of the navigation is on the component level, with base 0, having space up to 1999. The sub menus, however, raise to the top level under navigation (2000 - 2999). Where it is valid to be on top of the backdrop.

[Checkout the z-index strategy.](https://confluence.fti-group.com/display/PORTALS/z-index+strategy)

See:
```
.header-mobile-navigation {
  z-index: get-z-index("component", "header"); // 10
}

.backdrop {
  z-index: get-z-index("component", "backdrop"); // 1
}

.header-sub-menu-container {
    z-index: get-z-index("navigation", "header-sub-menu-desktop"); // 2900
  }
// It was 1 before, but since the DOM element comes after the backdrop, it would still go over the backdrop.

```

The whole reason to implement this strategy is to give more transparency. Also communicating it to teams developing external micro services, that we implement into the portal. The other reason is, we have better possibilities placing elements on the z-level. Very few situations shall restrict us.